diff --git a/node_modules/gatsby-theme-conference/src/components/mcs.js b/node_modules/gatsby-theme-conference/src/components/mcs.js
index afd28f8..d791b43 100644
--- a/node_modules/gatsby-theme-conference/src/components/mcs.js
+++ b/node_modules/gatsby-theme-conference/src/components/mcs.js
@@ -12,7 +12,7 @@ export default ({
     sx={{ py: 5 }}>
     <Container>
       <SectionHeading>
-        MCs
+        MC
       </SectionHeading>
       <FlexList
         sx={{
diff --git a/node_modules/gatsby-theme-conference/src/components/schedule-item.js b/node_modules/gatsby-theme-conference/src/components/schedule-item.js
index c3eb5b4..b485676 100644
--- a/node_modules/gatsby-theme-conference/src/components/schedule-item.js
+++ b/node_modules/gatsby-theme-conference/src/components/schedule-item.js
@@ -1,5 +1,6 @@
 /** @jsx jsx */
 import { jsx, Styled, Flex } from 'theme-ui'
+import paragraphs from 'lines-to-paragraphs'
 
 export default ({
   date,
@@ -13,7 +14,7 @@ export default ({
       flexWrap: [ 'wrap', 'nowrap' ],
       alignItems: 'baseline',
     }}>
-    <div
+    {/* <div
       sx={{
         flex: 'none',
         width: [ '50%', 128 ],
@@ -24,7 +25,7 @@ export default ({
         }}>
         {time}
       </Styled.h3>
-    </div>
+    </div> */}
     <div
       sx={{
         flex: 'none',
@@ -41,8 +42,8 @@ export default ({
     <div sx={{ width: '100%' }}>
       <Styled.p>
       </Styled.p>
-      <Styled.p>
-        {description}
-      </Styled.p>
+      <Styled.p
+        dangerouslySetInnerHTML={{ __html: paragraphs(description) }}
+      />
     </div>
   </Flex>
diff --git a/node_modules/gatsby-theme-conference/src/components/schedule.js b/node_modules/gatsby-theme-conference/src/components/schedule.js
index d0506b6..204100b 100644
--- a/node_modules/gatsby-theme-conference/src/components/schedule.js
+++ b/node_modules/gatsby-theme-conference/src/components/schedule.js
@@ -17,13 +17,13 @@ export default ({
       sx={{ py: 5 }}>
       <Container>
         <SectionHeading>
-          Schedule
+          Talks
         </SectionHeading>
         {Object.keys(days).map(date => (
           <div key={date}>
-            <Styled.h3>
+            {/* <Styled.h3>
               {date}
-            </Styled.h3>
+            </Styled.h3> */}
             <List>
               {days[date].map(item => (
                 <li key={item.time}>
diff --git a/node_modules/gatsby-theme-conference/src/components/speaker-card.js b/node_modules/gatsby-theme-conference/src/components/speaker-card.js
index 111dc21..a949321 100644
--- a/node_modules/gatsby-theme-conference/src/components/speaker-card.js
+++ b/node_modules/gatsby-theme-conference/src/components/speaker-card.js
@@ -7,6 +7,7 @@ import {
 import BackgroundImage from './background-image'
 import Card from './card'
 import IconLink from './icon-link'
+import paragraphs from 'lines-to-paragraphs'
 
 export default ({
   id,
@@ -40,9 +41,9 @@ export default ({
     <Styled.p
       sx={{
         mb: 0,
-      }}>
-      {bio}
-    </Styled.p>
+      }}
+      dangerouslySetInnerHTML={{ __html: paragraphs(bio) }}
+    />
     <Flex mx={-2}>
       {twitter && (
         <IconLink href={twitter}>
diff --git a/node_modules/gatsby-theme-conference/src/components/speakers.js b/node_modules/gatsby-theme-conference/src/components/speakers.js
index 3c7f188..c15d8bd 100644
--- a/node_modules/gatsby-theme-conference/src/components/speakers.js
+++ b/node_modules/gatsby-theme-conference/src/components/speakers.js
@@ -19,7 +19,7 @@ export default ({
       </SectionHeading>
       <FlexList
         sx={{
-          justifyContent: 'center',
+          justifyContent: 'left',
         }}>
         {speakers.map(speaker => (
           <SpeakerCard
diff --git a/node_modules/gatsby-theme-conference/src/components/sponsors.js b/node_modules/gatsby-theme-conference/src/components/sponsors.js
index 5ed854a..ceb99e0 100644
--- a/node_modules/gatsby-theme-conference/src/components/sponsors.js
+++ b/node_modules/gatsby-theme-conference/src/components/sponsors.js
@@ -2,33 +2,53 @@
 import { jsx, Styled, Container } from 'theme-ui'
 import SectionHeading from './section-heading'
 import FlexList from './flex-list'
-import Card from './card'
+import SponsorCard from './sponsor-card'
+import groupBy from 'lodash.groupby'
+
 
 export default ({
   sponsors = []
-}) =>
-  <div
-    id='sponsors'
-    sx={{ py: 5 }}>
-    <Container>
-      <SectionHeading>
-        Sponsors
-      </SectionHeading>
-      <FlexList
-        sx={{
-          justifyContent: 'center',
-        }}>
-        {sponsors.map(sponsor => (
-          <Card as='li' key={sponsor.id}>
-            <Styled.a
-              title={sponsor.name}
-              href={sponsor.url}>
-              <Styled.img
-                src={sponsor.logo}
-              />
-            </Styled.a>
-          </Card>
+}) => {
+  const tiers = groupBy(sponsors, 'tier')
+  console.log(tiers);
+  return(
+    <div
+      id='sponsors'
+      sx={{ py: 5 }}>
+      <Container>
+        <SectionHeading>
+          Sponsors
+        </SectionHeading>
+        {Object.keys(tiers).map(tier => (
+          <div key={tier}>
+            <Styled.h2>
+              {tier}
+            </Styled.h2>
+            <FlexList
+              sx={{
+                justifyContent: 'left',
+              }}
+            >
+              {tiers[tier].map(sponsor => (
+                // <Card as='li' key={sponsor.id}>
+                //   <Styled.a
+                //     title={sponsor.name}
+                //     href={sponsor.url}>
+                //     <Styled.img
+                //       src={sponsor.logo}
+                //     />
+                //   </Styled.a>
+                // </Card>
+                <SponsorCard
+                  key={sponsor.id}
+                  as='li'
+                  {...sponsor}
+                />
+              ))}
+            </FlexList>
+          </div>
         ))}
-      </FlexList>
-    </Container>
-  </div>
+      </Container>
+    </div>
+  )
+          }
diff --git a/node_modules/gatsby-theme-conference/src/components/venue.js b/node_modules/gatsby-theme-conference/src/components/venue.js
index 69d899e..f23a825 100644
--- a/node_modules/gatsby-theme-conference/src/components/venue.js
+++ b/node_modules/gatsby-theme-conference/src/components/venue.js
@@ -1,6 +1,8 @@
 /** @jsx jsx */
 import { jsx, Styled, Flex, Container } from 'theme-ui'
 import SectionHeading from './section-heading'
+import paragraphs from 'lines-to-paragraphs'
+
 
 export default ({
   title,
@@ -52,12 +54,12 @@ export default ({
           <Styled.p
             sx={{
               my: 0
-            }}>
-            {description}
-          </Styled.p>
+            }}
+            dangerouslySetInnerHTML={{ __html: paragraphs(description) }}
+          />
           <div>
-            <Styled.a href={mapURL}>View Map</Styled.a>
-            {' | '}
+            {/* <Styled.a href={mapURL}>View Map</Styled.a>
+            {' | '} */}
             <Styled.a href={url}>Website</Styled.a>
           </div>
         </div>
diff --git a/node_modules/gatsby-theme-conference/src/layouts/schedule.js b/node_modules/gatsby-theme-conference/src/layouts/schedule.js
index bce310c..1da9221 100644
--- a/node_modules/gatsby-theme-conference/src/layouts/schedule.js
+++ b/node_modules/gatsby-theme-conference/src/layouts/schedule.js
@@ -18,13 +18,13 @@ export default ({
           sx={{
             mb: 5,
           }}>
-          Schedule
+          Talks
         </Styled.h1>
         {Object.keys(days).map(date => (
           <div key={date}>
-            <Styled.h3>
+            {/* <Styled.h3>
               {date}
-            </Styled.h3>
+            </Styled.h3> */}
             <List>
               {days[date].map(item => (
                 <li key={item.time}>
diff --git a/node_modules/gatsby-theme-conference/src/use-site-metadata.js b/node_modules/gatsby-theme-conference/src/use-site-metadata.js
index cca40bf..810ed69 100644
--- a/node_modules/gatsby-theme-conference/src/use-site-metadata.js
+++ b/node_modules/gatsby-theme-conference/src/use-site-metadata.js
@@ -8,7 +8,9 @@ export default () => {
           title
           description
           twitter
-          instagram
+          instagram,
+          youtube,
+          facebook
         }
       }
     }
